ostd/src/mm/page_prop.rs

```diff
@@ -96,6 +96,22 @@ pub enum CachePolicy {
     Writeback,
 }
 
+#[cfg(target_arch = "x86_64")]
+impl CachePolicy {
+    /// Returns the `IA32_PAT` memory type encoding for this cache policy.
+    ///
+    /// Reference: Intel 64 and IA-32 Architectures SDM Vol. 3A, Table 11-10.
+    pub(crate) const fn to_ia32_pat_memory_type_entry(self) -> u8 {
```

Comment:
This should be moved to the x86 module? I suspect it should live inside `configure_pat`...
```rust
pub(crate) fn configure_pat() {
    fn cache_policy_to_pat_entry(cache_policy: CachePolicy) -> u8 { ... }
}
```